src/node_metadata.cc

```diff
@@ -11,6 +11,7 @@
 
 #if HAVE_OPENSSL
 #include <openssl/opensslv.h>
+#include <openssl/quic.h>
```

Comment:
I'm guessing this would prevent us linking (even dynamically) against a non-quic-forked OpenSSL?

Reply:
Ah, good point and something that I did not think of. I'll need to take a closer look how we can work around that. I just noticed this change today.

Reply:
I've added a commit (actually just amended) with a suggestion for handling this header.